File: CampingNeretva/UI/campingneretva_desktop/lib/models/user_preference_model.dart
class UserPreferenceModel {
  final int numberOfPeople;
  final bool hasSmallChildren;
  final bool hasSeniorTravelers;
  final String carLength;
  final bool hasDogs;

  UserPreferenceModel({
    required this.numberOfPeople,
    required this.hasSmallChildren,
    required this.hasSeniorTravelers,
    required this.carLength,
    required this.hasDogs,
  });

  factory UserPreferenceModel.fromJson(Map<String, dynamic> json) {
    return UserPreferenceModel(
      numberOfPeople: json['numberOfPeople'],
      hasSmallChildren: json['hasSmallChildren'],
      hasSeniorTravelers: json['hasSeniorTravelers'],
      carLength: json['carLength'],
      hasDogs: json['hasDogs'],
    );
  }
}

File: CampingNeretva/CampingNeretva.Model/Models/UserPreferenceModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model.Models
{
    public class UserPreferenceModel
    {
        public int UserPreferenceId { get; set; }
        public int UserId { get; set; }
        public int NumberOfPeople { get; set; }
        public bool HasSmallChildren { get; set; }
        public bool HasSeniorTravelers { get; set; }
        public string CarLength { get; set; }
        public bool HasDogs { get; set; }
    }
}
